mantidproject/mantid
# Adding maximum entropy method to Frequency Domain Analysis GUI

The MaxEnt algorithm has been added to the new Muon GUI (Frequency Domain Analysis, see #20205). This involved adding a menu bar that can be used to select the method (FFT or MaxEnt) along with an implementation of the MaxEnt GUI. These changes required some minor restructuring of the code. 

**To test:**
0. Open muon analysis (custom interface) and load some data (the current run will do). Go to the settings tab and bin the data (10) 

1. Open the Frequency Domain analysis GUI. 
2. Use the drop down menu to switch between the two interfaces. 
3. In the MaxEnt interface the data selection should automatically be loaded in the workspace dropdown menu.
4. Try various settings and run MaxEnt. The MaxEnt algorithm is expensive so if you use the Raw data (tick box) then it will take a long time.



Fixes #20233 
**Release Notes** 

The documentation will be updated in #20312
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
